modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -1835,6 +1835,34 @@ return view.extend({
 						o.placeholder = '201';
 						o.rmempty = true;
 
+						if (L.hasSystemFeature('hostapd', 'ocv') || L.hasSystemFeature('wpasupplicant', 'ocv')) {
+							o = ss.taboption('encryption', form.ListValue, 'ocv', _('Operating Channel Validation'), _("Note: Workaround mode allows a STA that claims OCV capability to connect even if the STA doesn't send OCI or negotiate PMF."));
+							o.value('0', _('Disabled'));
+							o.value('1', _('Enabled'));
+							o.value('2', _('Enabled (workaround mode)'));
+
+							add_dependency_permutations(o, { mode: ['ap', 'ap-wds', 'sta', 'sta-wds'], encryption: ['psk-mixed', 'psk2', 'sae', 'sae-mixed', 'wpa2', 'wpa3', 'wpa3-mixed', 'owe'] });
+
+							o.validate = function(section_id, value) {
+								var mfpopt = this.section.children.filter(function(o) { return o.option == 'ieee80211w' })[0],
+								mfpval = mfpopt.formvalue(section_id);
+
+								if ((value == '1' || value == '2') && mfpval == '0') {
```

Comment:
Doesn't it work just to add a depends on `mfpval`?

It returns a string which mentions values whose names might change or not be translated.

Reply:
Sorry, I'm very much a noob so please bear with me. What do you mean by adding a depends on `mfpval`? I looked at the other uses of the validate method in the script and they all return strings in some form.

Reply:
https://openwrt.github.io/luci/jsapi/LuCI.form.ListValue.html#depends

Reply:
I can remove that part of the validator function and just add the 802.11w dependency to the add_dependency_permutations part of the code:

```javascript
add_dependency_permutations(o, { mode: ['ap', 'ap-wds', 'sta', 'sta-wds'], encryption: ['psk-mixed', 'psk2', 'sae', 'sae-mixed', 'wpa2', 'wpa3', 'wpa3-mixed', 'owe'], ieee80211w: ['1', '2']});
```

The reason I had that validator function in the first place is because I thought it would be a good idea to alert the user about the requirement of 802.11w for ocv, but I think it's ok to drop this to avoid the string translation/renaming issue.

Reply:
I don't trust that function which is why I recommended just depends. I'm not certain that the permutations function does what it should, until one has tested every variant. We had problems that meant we had to revert recently because of dep permutations. For a PR which was here for 3 years...

Reply:
Ah, now I get it. I just had a look at the dependencies of 802.11w and realised that it covered what I needed. I've now changed the code to use depends calls. Thanks!